WcaOnRails/app/helpers/documents_helper.rb

```diff
@@ -2,9 +2,9 @@
 
 module DocumentsHelper
   def documents_list(directory)
-    safe_join Dir.glob("#{Rails.root}/public/documents/#{directory}/*.pdf")
+    safe_join Dir.glob("#{Rails.root}/public/#{directory}/*.pdf")
```

Comment:
Given everything is still nested within `documents` I believe this is not necessary anymore :)

Reply:
Reverted